pkg/entities/conflict_response.go

```diff
@@ -0,0 +1,8 @@
+package entities
+
+// ConflictResponse represents the HTTP code 409 Conflict
+type ConflictResponse struct{}
+
+func (err ConflictResponse) Error() string {
+	return "Error 409"
```

Comment:
The `ErrEntityExists` error could store entity details (such as its id/name/email) in a wat that you can create a descriptive error message here. Also, It's [recommended](https://github.com/golang/go/wiki/CodeReviewComments#error-strings) not to start the message with capital letters.